src/main/java/net/imglib2/type/label/AbstractLabelMultisetLoader.java

```diff
@@ -18,9 +18,6 @@
  * @author Neil Thistlethwaite
  */
 
-// TODO this currently only works because it resides in the same package as LongMappedAccessData (albeit in a different artifact).
-// This is very fragile!
-
```

Comment:
This does not seem to be problematic anymore. The protected `data` field of `LongMappedAccessData` was used in the N5 cache loader class (which was moved to a different package). But, the same field is exposed via the public `getData()` method, so the change was easy and it does not break anything.